src/substrait/gen/json/simple_extensions.py

```diff
@@ -1,6 +1,6 @@
 # generated by datamodel-codegen:
 #   filename:  simple_extensions_schema.yaml
-#   timestamp: 2025-06-06T08:43:35+00:00
+#   timestamp: 2025-10-24T17:55:30+00:00
```

Comment:
I guess we could disable adding the timestamp in `Makefile` by adding the `--disable-timestamp` flag to make repeated `make codegen-extensions` runs produce reproducible outputs